qt/python/mantidqt/widgets/memorywidget/memorypresenter.py

```diff
@@ -7,36 +7,29 @@
 #  This file is part of the mantidqt package
 #
 #
-from qtpy.QtCore import QTimer
+from threading import Timer
 
 from ..memorywidget.memoryinfo import get_memory_info
+from ...utils.asynchronous import set_interval
 
-TIME_INTERVAL_MEMORY_USAGE_UPDATE = 2000  # in ms
+TIME_INTERVAL_MEMORY_USAGE_UPDATE = 2.000  # in s
 
 
 class MemoryPresenter(object):
     """
-    Gets system memory usage information and passes it to
-    the memory view
-    This happens at the beginning as well as every
-    TIME_INTERVAL_MEMORY_USAGE_UPDATE (ms) using QTimer
-    Besides, sets the style of the memory(progress) bar
-    at the beginning
+    Gets system memory usage information and passes it to the memory view this happens at the beginning as well as
+    every TIME_INTERVAL_MEMORY_USAGE_UPDATE (s) using threading.Timer; Also, sets the style of the memory(progress) bar
+    on construction.
     """
     def __init__(self, view):
+        import pydevd_pycharm
```

Comment:
Some debugging statements left I think :)

Reply:
Ack I un-stashed the changes as I was comparing the previous implementation and this one, and just committed. Thanks for catching that!